<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>askforleave</class>
 <widget class="QWidget" name="askforleave">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>566</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCalendarWidget" name="LeaveCalendarWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>551</width>
     <height>289</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="gridVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="leaveData">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>350</y>
     <width>551</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QTreeWidget" name="Leavetree">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>171</width>
     <height>381</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>171</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>查询时间段</string>
   </property>
   <widget class="QCheckBox" name="QchkSTime">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>始</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="QchkETime">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>末</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="QdateSTime">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="QdateETime">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
